src/oca_pre_commit_hooks/checks_odoo_module_xml.py

```diff
@@ -23,7 +23,7 @@ def _hasclass(context, *cls):
 
 class ChecksOdooModuleXML(BaseChecker):
     xpath_deprecated_data = etree.XPath("/odoo[count(./*) < 2]/data|/openerp[count(./*) < 2]/data")
-    xpath_oe_structure_woid = etree.XPath("//*[hasclass('oe_structure') and not(@id)]")
+    xpath_oe_structure_woid = etree.XPath("//*[hasclass('oe_structure') and not(@id) and count(./*)=0]")
```

Comment:
After reading [this](https://github.com/odoo/odoo/blob/25edcc89273c389f318c434ce9a9e0b531b8e599/addons/web_editor/models/ir_ui_view.py#L41), I noticed the provided ID should actually contain `oe_structure`. So, the suggestion shouldn't be a suggestion but a rule.

CC @rolandojduartem

Reply:
Yes, I have added it in the last commit of this MR. Will change the title and update description. Just need to confirm the thing about `t-attf-id` and `id="wrap"` (discussion on this is in the issue)